draft-linker-adem-wg-adem-core.md

```diff
@@ -130,11 +134,16 @@ Consequently, this section also specifies the semantics of identifiers.
 ### Entity Identifiers
 
 Entities that can be technically marked as protected are processes, or data in non-volatile storage.
+
+TODO(djackson) - ??? 
+
 In the following, we describe how such entities are identified.
 Entities are identified by *entity identifiers* (EIs).
 Entity identifiers closely resemble Uniform Resource Identifiers (URIs) as specified in {{!RFC3986}}.
 However, to limit their scope, we do not follow the specification of URIs and instead define our own syntax.
 
+//TODO(djackson) - In what ways do we diverge? Aren't we aiming for a strict subset? 
```

Comment:
Using wildcards is not in spec. You cannot use `*` in URIs afaik.

Reply:
I guess there's a standard for x509 certificate validation that allows it though? E.g. wildcard domains.